meta/sai_meta.cpp

```diff
@@ -1824,14 +1824,6 @@ sai_status_t meta_port_remove_validation(
         return SAI_STATUS_SUCCESS;
     }
 
-    if (object_reference_count(port_id) != 0)
```

Comment:
why this is removed ?

Reply:
Isn't the line 1917 does the same thing?

Reply:
yes, but i have a big changes and this code would need to be pulled in to my new branch: https://github.com/Azure/sonic-sairedis/compare/master...kcudnik:bigrebase?expand=1 and this branch will be rebased to master this or next week, and it still contains this code but in different files so you could then make this  PR on new code

Reply:
how difficult is it to pull this into you branch?

Reply:
OK. I got it. Let me wait and make these changes when your code is merged.

Reply:
please rebase, new origin/master is laready present